### PR TITLE
Concrete syntax

### PR DESCRIPTION
**Pull request by laurayuwen**

### CODE REVIEW

**Comment by laurayuwen** [comment]
@andreistefanescu Please review

File: src/common/core-functions.k [hunk 1]
```diff
@@ -231,6 +231,11 @@ syntax ClassBodyDecList ::= addElementToClassBodyDecListEnd (ClassBodyDecList, C
 rule addElementToClassBodyDecListEnd(.ClassBodyDecList, P:K) => 'ClassBodyDecList(P,, .ClassBodyDecList)
 rule addElementToClassBodyDecListEnd('ClassBodyDecList(Hd:K,,Tl:K), P:K) => 'ClassBodyDecList(Hd,, addElementToClassBodyDecListEnd(Tl,P))
 
+syntax ClassBodyDecList ::= appendClassBodyDecList(ClassBodyDecList, ClassBodyDecList) [function]
+rule appendClassBodyDecList(AExps:ClassBodyDecList, (Hd:ClassBodyDec Tl:ClassBodyDecList)) =>
```
Comment (andreistefanescu) [inline src/common/core-functions.k]: maybe name this variable something like ```CBDs``` or ```L```

**Comment by andreistefanescu** [comment]
lgtm

File: src/common/core-functions.k (hunk 1 above)
Comment (laurayuwen) [inline src/common/core-functions.k]: done

**Pull request merged**